### PR TITLE
update primary name change article

### PR DESCRIPTION
**Pull request by stephamon**
Add self-service option for MyRackspace portal

JIRA 2308

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-6098ae5395/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f71a1bc7e4/how-to/).

File: content/general/change-to-new-primary-contact.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ title: Change an account's primary contact
 type: article
 created_date: '2018-03-13'
 created_by: Rackspace Support
-last_modified_date: '2018-05-23'
-last_modified_by: Nate Archer
+last_modified_date: '2018-08-20'
```
Comment (RackerKate) [inline content/general/change-to-new-primary-contact.md]: Should this be 08-22?

File: content/general/change-to-new-primary-contact.md [hunk 2]
```diff
@@ -31,8 +33,18 @@ You can download this form directly from the Cloud Control Panel or MyRackspace
 7. Attach your saved Primary Contact Change Form to the ticket.
 8. Click **Create Ticket**.
 
+#### MyRackspace Control Panel self-service option
+
+Use the following steps to change the primary contact for your account:
+
+1. In the top navigation bar, click **Account > User List**.
+2. In the **Account** sub-navigaton bar, click **Account Settings**.
+3. Click the pencil icon in the Primary Contact Information section to change the primary contact for your account.
```
Comment (RackerKate) [inline content/general/change-to-new-primary-contact.md]: Should "Primary Contact Information" be bold?

File: content/general/change-to-new-primary-contact.md [hunk 3]
```diff
@@ -31,8 +33,18 @@ You can download this form directly from the Cloud Control Panel or MyRackspace
...
+
 #### Cloud Control Panel
 
+Use the following steps to submit a ticket to change the primary contact for your account:
+
 1. In the upper-right corner of the Cloud Control Panel, click **Account** > **Docs and Forms**.
```
Comment (RackerKate) [inline content/general/change-to-new-primary-contact.md]: Does the whole thing usually get bolded, including the >?

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-07752b45a1/how-to/)

**Pull request merged**